streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java

```diff
@@ -47,7 +59,26 @@ public class MeteredTimestampedKeyValueStore<K, V>
         super(inner, metricScope, time, keySerde, valueSerde);
     }
 
-
+    @SuppressWarnings("rawtypes")
+    private final Map<Class, StoreQueryUtils.QueryHandler> queryHandlers =
+            mkMap(
+                    mkEntry(
+                            RangeQuery.class,
+                            (query, positionBound, config, store) -> runRangeQuery(query, positionBound, config)
```

Comment:
I think we need to overwrite `runRangeQuery` in this class, and change the behavior to unwrap the `ValueAndTimestamp` we get from the inner store, to only return "plain V". (Same for `runKeyQuery`.)

Reply:
ok